model_analyzer/config/run_config/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
What about moving files to `config/run` and `config/input`? instead of run_config and input_config.